tools/build_steps.sh

```diff
@@ -181,11 +181,14 @@ EOF
             export DYLD_LIBRARY_PATH=/usr/local/lib:$DYLD_LIBRARY_PATH
             ;;
         *-s390x)
+            # The TargetList.txt has only ZARCH_GENERIC, Z13, Z14. Not worth
+            # messing with dynamic lists and targets.
             local bitness=64
             ;;
         *-ppc64le)
             local bitness=64
             local target="POWER8"
+            local dynamic_list="POWER8 POWER10"
```

Comment:
```suggestion
```

ppc64le does not support dynamic list

Reply:
The [README](https://github.com/OpenMathLib/OpenBLAS/blob/f6df9bebbb4259aa61ab5634c0f1269fb152cc0e/README.md?plain=1#L263) seems to imply that DYNAMIC_LIST is supported
> For **POWER**, the list encompasses POWER6, POWER8 and POWER9. POWER10 is additionally available if a sufficiently recent compiler is used for the build.

Reply:
Strange. Is there an open OpenBLAS issue about the missing `DYNAMIC_LIST` override of `DYNAMIC_CORE`?